code/ui/blocks/src/blocks/Story.stories.tsx

```diff
@@ -51,7 +51,7 @@ export const SimpleSizeTest: Story = {
         style={{
```

Comment:
When changing the stories from Boolean Control to Button I missed the one that uses `id` above on line 33. could you change it?

Reply:
Done. Although should this be in an internal story file do you think?

Reply:
> @JReinhold -- what shall we do with the unaccepted changes here? They are stories that are currently broken due to the discussion here: [#20530 (comment)](https://github.com/storybookjs/storybook/pull/20530#issuecomment-1375104177) -- shall we accept them as is or disable them for now?

Let's accept them, otherwise they'll spring up in other unrelated PRs and confuse the team.